packages/parser/src/utils/get-media-query.ts

```diff
@@ -21,7 +21,12 @@ export function getMediaQuery(breakpoints?: Dict, mapper = toMediaQuery) {
 
   const asArray = objectKeys(_breakpoints)
     .map((key) => _breakpoints[key])
-    .sort((a, b) => a - b)
+    .sort((a, b) => {
+      const regex = /^\d+/
```

Comment:
I'm unaware under which circumstances this is called so please forgive me if this is wrong, but but I'd assume this is performance sensitive so we might be able to gain a small win here by having the regex outside of the compare function, or even extract the whole sort-function to its own definition?